mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -114,7 +115,12 @@ object MLUtils {
 
   // Convenient methods for `loadLibSVMFile`.
 
-  @deprecated("use method without multiclass argument, which no longer has effect", "1.1.0")
```

Comment:
I think we need to leave the `@deprecated` outside the JavaDoc as a Scala annotation to make it work in Scala.